api/src/test/java/uk/gov/hmcts/auth/provider/service/api/componenttests/TestSupportLeaseComponentTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.auth.provider.service.api.componenttests;
+
+import org.junit.Test;
+
+public class TestSupportLeaseComponentTest extends ComponentTestBase {
+
+    @Test
+    public void should_generate_jwt_for_service() throws Exception {
+        String serviceName = "some_arbitrary_service_name";
+
+        String token =
+            scenario
+                .given()
+                .when()
+                .testSupportLease(serviceName)
+                .then()
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        scenario
+            .given()
+            .when()
+            .details("Bearer " + token)
```

Comment:
surprisingly, this method takes a header value, not a token as the method signature would suggest:
```
details(String token)
```
O_o
Will refactor on a separate occasion.